go/storage/mkvs/node/key.go

```diff
@@ -26,12 +26,6 @@ func (k Key) MarshalBinary() (data []byte, err error) {
 	return
 }
 
-// UnmarshalBinary decodes a binary marshaled key including the length in bytes.
-func (k *Key) UnmarshalBinary(data []byte) error {
```

Comment:
I would not remove this function as it is public and may be used in other packages, and I think it is nice that `MarshalBinary` is accompanied by `UnmarshalBinary`. It doesn't hurt anyone to have it there.